src/snippets/muiSvgIcon.ts

```diff
@@ -0,0 +1,8 @@
+import snip from '../snip'
+
+export const description = 'Material-UI <SvgIcon>'
+export const body = snip`
+<SvgIcon {...props}>
```

Comment:
Seems like this is for a custom component that just `return <SvgIcon {...props}>...`, maybe we should make the snippet be a whole component definition instead?